<launch>
    <arg name="usb_port"    default="/dev/ttyUSB0"/>
    <arg name="dxl_baud_rate"   default="57600"/>    
    <param name="dynamixel_info"    value="$(find my_dynamixel_workbench)/cfg/SerialRobot.yaml"/>
    <param name="stateRecordPath" value="$(find my_dynamixel_workbench)/data/state.xls"/>
    <node name="multi_motors_demo" pkg="my_dynamixel_workbench" type="multi_motors_demo" 
            output="screen"    args="$(arg usb_port) $(arg dxl_baud_rate)" >
        <rosparam>
        </rosparam>
    </node>
</launch>
